### PR TITLE
add custom promtail image

### PR DESCRIPTION
**Pull request by robert-oleynik**

### CODE REVIEW

File: promtail/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM grafana/promtail:main
```
Comment (normanrz) [inline promtail/Dockerfile:1]: Should we pin a version here?

Comment (robert-oleynik) [reply promtail/Dockerfile]: Used the version specified in: https://artifacthub.io/packages/helm/grafana/promtail (AppVersion: 3.0.0)

**Comment by normanrz** [comment]
You need to add these to the CI config https://github.com/scalableminds/dockerfiles/blob/master/.github/workflows/main.yml#L9

File: promtail/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Usage
+
+Run using docker:
+
+```sh
+docker run -v /var/run/docker.sock:/var/run/docker.sock \
+    -e LOKI_PUSH_URL="<domain>/loki/api/v1/push" \
+    -e LOKI_PUSH_USER="<user>" \
+    -e LOKI_PUSH_PASSW="<password>" \
```
Comment (normanrz) [inline promtail/README.md:11]:
```suggestion
    -e LOKI_PUSH_PASSWORD="<password>" \
```
less confusing, imo

**Review by normanrz — APPROVED** [review]

File: promtail/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+This will scrape all containers with label `gather.logs=true`.
+
+## Configuration
+
+Environment Variables:
+
+| Name | Description |
+|------|-------------|
+| `LOKI_PUSH_URL` | URL to push to (e.g. `<domain>/loki/api/v1/push`) |
+| `LOKI_PUSH_USER` | User for Basic Auth |
```
Comment (normanrz) [inline promtail/README.md:26]: In metrics-pusher, this is called AUTH_USER and AUTH_PASSWORD. Let's make it consistent

**Pull request merged**